src/XliffTasks/build/XliffTasks.targets

```diff
@@ -30,14 +30,28 @@
     <XlfIntermediateOutputPath Condition="'$(XlfIntermediateOutputPath)' == ''">$(IntermediateOutputPath)$(MSBuildProjectName).xlf\</XlfIntermediateOutputPath>
     <_XlfSourceList>$(XlfIntermediateOutputPath)XlfSources.txt</_XlfSourceList>
     <_UpdateXlfLastRun>$(XlfIntermediateOutputPath)UpdateXlf.lastrun</_UpdateXlfLastRun>
+  </PropertyGroup>
 
+  <!--
+    Suppress execution in certain contexts
+  -->
+  <PropertyGroup>
+    <!-- Suppress if not XLF localization not enabled -->
```

Comment:
Nit: I'd add `<_SuppressUpdateXlfOnBuild>false</_SuppressUpdateXlfOnBuild>` in front.

Reply:
Thanks. Updated.